### ISSUE TITLE
Dropping chapter heading lines

### ISSUE THREAD

**Issue by michardy** [michardy/note-digest#12]

**Comment by michardy** [comment]
The line ratio that was set 0.07 was too small.  Why is this a problem

for reference:
``` Rust
Some(ImgBlob {
	blob_type:
		if
			(bitmap[0].len() as f32 / bitmap.len() as f32) > LINE_RATIO// &&
				//bitmap[0].len() > 20
		{1} else {0},
	top_left: [left, top],
	bottom_right: [left+bitmap[0].len(), top+bitmap.len()],
	bitmap: bitmap
})
```
Where `LINE_RATIO` is 0.07

**Comment by michardy** [comment]
Setting the constant back to 10.0 recreates an earlier error where it is sometimes too sensitive

**Comment by michardy** [comment]
This error is still a problem
```
◑: Dividing by chapterthread 'main' panicked at 'Output Generation: Error empty content object', src/main.rs:468:3
note: Run with `RUST_BACKTRACE=1` for a backtrace.
zsh: abort (core dumped)  target/release/notedigest
```

**Comment by michardy** [comment]
Closed in favor of #13

**Comment by michardy** [comment]
Summary of fix:

changed threshold and added a minimum line length to prevent "t" crosses from becoming lines
